src/main/java/core/basesyntax/Entry.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Objects;
+
+public class Entry<K,V> {
+    private K key;
+    private V value;
+
+    public Entry(K key, V value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public K getKey() {
+        return this.key;
+    }
+
+    public V getValue() {
+        return this.value;
+    }
+
+    public Entry<K, V> setKey(K key) {
+        this.key = key;
+        return this;
+    }
+
+    public Entry<K, V> setValue(V value) {
+        this.value = value;
+        return this;
+    }

```

Comment:
According to the checklist, don't use getters and setters in class Node. Since Entry class is similar to Node class and is an inner class, you should apply the same principle here. You can directly access the private fields of Entry within the MyHashMap class, so the getters and setters are redundant.

Reply:
but 'maven clean package' command forced me to make the key and value private

Reply:
getters and setters, are created due to 'maven clean package' recomendation,